js/src/carousel.js

```diff
@@ -244,16 +261,22 @@ const Carousel = (($) => {
     }
 
     _addEventListeners() {
+      const touchSupported = 'ontouchstart' in document.documentElement
       if (this._config.keyboard) {
         $(this._element)
           .on(Event.KEYDOWN, (event) => this._keydown(event))
       }
 
+      if (touchSupported) {
+        this.hammer.on(Event.SWIPELEFT, () => this.next())
```

Comment:
Isn't here a check quired if hammer support is enabled or the hammer reference exists to prevent a reference error? 🤔

Reply:
Yep that's true nice catch 👍